docs/symptom-survey/survey-files.md

```diff
@@ -26,49 +26,75 @@ where the data is hosted.
+We provide two types of data files, daily and monthly. Users who need the most
+up-to-date data should use the daily files, while those who want to conduct
+retrospective analyses using many months of data may find the monthly files more
+convenient.
 
-	{YYYY_mm}.tar
+### Daily Files
 
-Incremental files:
+Each day, we write CSV files with names following this pattern:
 
 	cvid_responses_{for}_recordedby_{recorded}.csv.gz
 
 Dates in incremental filenames are of the form `YYYY_mm_dd`. `for` refers to the
 day the survey response was started, in the Pacific time zone (UTC -
 7). `recorded` refers to the day survey data was retrieved; see the [lag
-policy](#lag-policy) for more details.
+policy](#lag-policy) for more details. Each file is compressed with gzip, and
+the standard `gunzip` command on Linux or Mac can decompress it.
 
 Every day, we write response files for all recent days of data, with today's
-`recorded` date. You need only load the most recent set of `recorded` files to
-obtain all survey responses; the older versions are available to track any
-changes in file formats or slight changes from late-arriving responses, as
-described in the [lag policy below](#lag-policy).
-
-## Loading Data Files
-
-As described above, one day of data may be reissued several times, if responses
-arrive late, file formats are changed, or errors in data processing are
-corrected. You need only load the latest version of each file.
+`recorded` date. For each `for` date, you need only load the most recent
+`recorded` file to obtain all survey responses; the older versions are available
+to track any changes in file formats or slight changes from late-arriving
+responses, as described in the [lag policy below](#lag-policy).
 
 For data users who use R to load and process data, we provide a [`get_survey_df`
 function](survey-utils.R) to read a directory of CSV files (such as those
 provided on the SFTP server), select the correct files, and read them into a
 single data frame for use.
 
+### Monthly Files
+
+Several days after the end of each month, we produce "rollup" files containing
+all survey responses from that month. These are in two forms.
+
+First, the monthly CSV files have filenames in the form
+
+    {YYYY}-{mm}.csv.gz
+
+and contain all valid responses for that month. These are produced from the
+daily files, by taking the data with the most recent `recordedby` date for each
+day of the month. They are compressed with gzip; the standard `gunzip` command
+on macOS or Linux can decompress them. (macOS can also decompress these files
+through Finder automatically; on Windows, free programs like
+[7-zip](https://www.7-zip.org/) can decompress gzip files.) Users doing
+historical analyses of the survey data should start with these files, since they
+provide the easiest way to get all the necessary data, without accidentally
+including duplicate results.
+
+Second, we produce monthly tarballs containing all the daily `.csv.gz` files for
```

Comment:
(monthly tarballs are just the most recent recordedby file for each date)

Reply:
fixed